packages/ember-cli-fastboot/test/fastboot-rendermode-config-test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
We can start writing tests involving fastboot / ember-cli-fastboot in https://github.com/ember-fastboot/ember-cli-fastboot/blob/2a6ed5d76b00da2106a9ba10f048aa2bef989a80/test-packages/integration-tests/test/basic-test.js.
But we can land this PR and do another follow up to move later.

Reply:
oh is this the new place that @kiwiupover has been working on? to be honest I wasn't sure and I just copy and pasted an existing test to get it to work. I would much prefer to do it the right way in this PR rather than fix it up in another one, I just don't exactly know how 😂

Reply:
You can move the fixture app to `test-packages`, then assert result in `test-packages/integration-tests/test/basic-test.js`. But don't worry I can get a test when we figure out how we want to name the config flag.